src/common/git/__tests__/index.ts

```diff
@@ -0,0 +1,72 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import {NodeProject, NodeProjectOptions} from 'projen/lib/javascript'
+import {synthSnapshot} from 'projen/lib/util/synth'
+import {addHusky, extendGitignore} from '..'
+
+describe('extendGitignore function', () => {
```

Comment:
@quesabe is this relevant to the PR?

Reply:
Actually the extendGitignore tests are not added here. I moved the tests for extendGitignore and added the husky tests to the same file (since they are now under the `git` folder). Locally in _git_ it was marked as a move, but somehow GitHub does not recognise it and shows a file deletion followed by a file creation.

Reply:
Interestingly for me GitHub does not show all this lines as new - it shows the file as modified with the `addHusky` tests added.